packages/components/src/components/drawer/drawer.jsx

```diff
@@ -39,7 +38,10 @@ const Drawer = ({
         }
     };
 
-    const is_mobile = isMobile();
+    // Note: Cannot use isMobile from @deriv/shared due to dimension change error
+    // TODO: Maybe we can fix isMobile in @deriv/shared
+    const isMobileDrawer = () => window.innerWidth < 768;
+    const is_mobile = isMobileDrawer();
```

Comment:
It's not a good solution to define a new `mobile_screen_size`. you should think about other solutions, e,g:
1- Use the  `isTablet` method to get the table size or define a new screen size in shared 
2- Ask the Design team to define the exact size which we should have the mobile view, 768px could have the mobile view instead of the Desktop view